back/spec/factories/custom_fields.rb

```diff
@@ -84,6 +84,22 @@
       end
     end
 
+    factory :custom_field_page do
+      title_multiloc do
+        {
+          'en' => 'Cycling survey'
+        }
+      end
+      description_multiloc do
+        {
+          'en' => 'This is a survey on your cycling habits.'
+        }
+      end
+      required { false }
+      input_type { 'page' }
+      enabled { true }
```

Comment:
Should we add model validations that page fields are always enabled and not required, or would that be overkill?

Reply:
I think it is overkill. We do not validate `enabled` for any survey form field. But as we discussed this morning, there is room for improvement.